docs/book/discovery.md

```diff
@@ -103,3 +103,101 @@ In both cases, OPA's configuration is hierarchically organized under the
 `discovery.name` value. If discovery is enabled, the `service` field in the
 `bundles`, `status`, `decision logs` plugins is optional and will default to one
 of the services from the discovery configuration.
+
+## Example
+
+Let's see an example of how the discovery feature can be used to dynamically configure an OPA to download one of two bundles based on an environment variable that the OPA was started with. Let's say the environment variable `region` indicates the region in which the OPA is running and it's value will decide the bundle to download.
+
+Below is a policy file which includes the bundle congfiguration.
+
+**example.rego**
+
+{%ace lang='python'%}
+package example
+
+discovery = {
+  "bundle": {
+    "name": bundle_name
+  }
+}
+
+rt = opa.runtime()
```

Comment:
Nit: I'd probably just make these into the body of the discovery rule. Either works.